docs/events/eboard-meetings.rst

```diff
@@ -0,0 +1,49 @@
+########
+Eboard Meetings
+########
+
+This page documents RITlug eboard meetings.
+Topics include room booking, preparation, and running of the meeting.
+
+
+*****************
+Meeting checklist
+*****************
+
+Pre-meeting
+===========
+
```

Comment:
Although heavily implied, there should be mention that all points of topic at each eboard meeting needs a corresponding issue to go alongside the topic, so that way we have documentation of the issue.